Scripts/Runtime/Entities/TaskDriver/TaskSet/Job/Wrapper/CDFEAccessWrapper.cs

```diff
@@ -5,15 +5,15 @@
 namespace Anvil.Unity.DOTS.Entities.TaskDriver
 {
     internal class CDFEAccessWrapper<T> : AbstractAccessWrapper
```

Comment:
Should we rename to `CDLAccessWrapper`? `ComponentLookupAccessWrapper`? `CDAccessWrapper`?

Based on the existing `DynamicBufferAccessWrapper` maybe `ComponentDataAccessWrapper` is the most appropriate.

Reply:
Yea, naming is tough. Regardless of what we decide let's make the change in a new PR so we don't clutter this one further.

I think I prefer Option 2 the most or leaving it as is. "Component Data From Entity" is still a valid description of the behaviour. 

To me, I'm not worried about implying sequential access. The usage of the reader/writer is the biggest hint that it's random access. Getting data off of an Entity key is clearly a lookup operation.

I vote either `ComponentDataReader` or `CDFEReader` with a preference to choose the latter and leave it as is. It's short and descriptive.

This isn't a hard position though and I'm open to other opinions.

Reply:
I've created the issue here: https://github.com/decline-cookies/anvil-unity-dots/issues/303

But I also agree, I'm cool with leaving as CDFE.